apps/drain-safe/src/__tests__/App.test.js

```diff
@@ -146,11 +148,15 @@ describe('<App />', () => {
   it('Shows a Warning icon when token transfer cost is higher than its current market value ', async () => {
     renderWithProviders(<App />);
 
+    await screen.findByText(/maker/i);
+
     const warningTooltip =
       /Beware that the cost of this token transfer could be higher than its current market value \(Estimated transfer cost: /i;
 
+    const tableRows = document.querySelectorAll('.MuiDataGrid-row');
```

Comment:
I think that this repeated line can be deleted

Reply:
yep!!, done